### PR TITLE
Makefile in addon generator

### PR DESCRIPTION
**Pull request by tiberiuichim**
I've added a Makefile in the addon generator, it provides the ability to spin up a Volto development instance from inside an addon.

### CODE REVIEW

File: packages/generator-volto/generators/addon/templates/Makefile.tpl [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# From: https://coderwall.com/p/izxssa/colored-makefile-for-golang-projects
+RED=`tput setaf 1`
+GREEN=`tput setaf 2`
+RESET=`tput sgr0`
+YELLOW=`tput setaf 3`
+
+project:
+	npm install -g yo
+	npm install -g @plone/generator-volto
+	npm install -g mrs-developer
```
Comment (sneridagh) [inline packages/generator-volto/generators/addon/templates/Makefile.tpl:16]: Globally?

**Review by sneridagh — APPROVED** [review]
LGTM, how about testing ;)

File: packages/generator-volto/generators/addon/templates/Makefile.tpl (hunk 1 above)
Comment (tiberiuichim) [reply packages/generator-volto/generators/addon/templates/Makefile.tpl:16]: It's based on Volto docs :) https://docs.voltocms.com/getting-started/install/#install-volto

**Comment by tiberiuichim** [comment]
Actually I've noticed that there's no "api" part in the generator. The whole backend setup needs to be removed from the Makefile.

File: packages/generator-volto/DEVELOP.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    cd volto/packages/generator-volto
+    yarn
+    npm link
+
+### Generate new project
+
+    yo @plone/volto my-volto-project --skip-install
+    cd my-volto-project
+
+### Generate new add-on based on remote template
```
Comment (avoinea) [inline packages/generator-volto/DEVELOP.md:15]: I think the right title here is `Generate new add-on`

File: packages/generator-volto/generators/addon/templates/Makefile.tpl (hunk 1 above)
Comment (avoinea) [reply packages/generator-volto/generators/addon/templates/Makefile.tpl:16]: I always install these globally 😄

**Review by avoinea — APPROVED** [review]

**Pull request merged**